pkg/cmd/server/origin/controller/build.go

```diff
@@ -7,8 +7,7 @@ import (
 
 	builddefaults "github.com/openshift/origin/pkg/build/admission/defaults"
 	buildoverrides "github.com/openshift/origin/pkg/build/admission/overrides"
-	buildclient "github.com/openshift/origin/pkg/build/client"
-	buildcontrollerfactory "github.com/openshift/origin/pkg/build/controller/factory"
+	buildcontroller "github.com/openshift/origin/pkg/build/controller/build"
```

Comment:
Eventually we want to push this wiring down into sub-repos, but it wasn't a primary objective and we don't have a solid interface yet.  Right now, the it would create really strange interdependencies.